packages/apputils/src/sessioncontext.tsx

```diff
@@ -1324,6 +1322,7 @@ export namespace SessionContext {
 export class SessionContextDialogs implements ISessionContext.IDialogs {
   constructor(options: ISessionContext.IDialogsOptions = {}) {
     this._translator = options.translator ?? nullTranslator;
+    this._settingRegistry = options.settingRegistry;
```

Comment:
I'm not sure where to get this from, in this it gets pass though from somewhere, 
https://github.com/jupyterlab/jupyterlab/blob/82051a9d0c6dc93677b0b66560aea48d71aaa043/packages/docmanager-extension/src/index.tsx#L929-L930